projects/i18n/tools/language.ts

```diff
@@ -1,11 +1,18 @@
-import {InjectionToken} from '@angular/core';
-import {Language} from '@taiga-ui/i18n/interfaces';
+import {inject, InjectionToken} from '@angular/core';
+import {TuiLanguage} from '@taiga-ui/i18n/interfaces';
 import {TUI_ENGLISH_LANGUAGE} from '@taiga-ui/i18n/languages';
 import {Observable, of} from 'rxjs';
 
-export const TUI_LANGUAGE = new InjectionToken<Observable<Language>>(
-    `Language for Taiga UI libraries i18n`,
+export const TUI_DEFAULT_LANGUAGE = new InjectionToken<Observable<TuiLanguage>>(
```

Comment:
Default token for default value of another token ? Do we really need it ?

Reply:
Yes, because we are override `TUI_LANGUAGE` by service and return fallback `TUI_DEFAULT_LANGUAGE` when storage is empty